scripts/seedsDB.js

```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
   // This file empties the risks collection and inserts the books below
   const riskSeed = [   
     {
-        title: "title1",
+        title: "London Eye",
         riskId: "riskId1",
-        description:"discription1",
+        description: "Annual maintenance",
         designDiscipline: "designDiscipline1",
         status: 2,
         location: {
-            lat: 52.505641,
-            lng: -1.923124,
+            lat: 50.353626,
+            lng: 1.6235,
           },
         likelihood:4,
         severity:4,
         risk:8,
-        projectId:"5f4eec89cc853f32ec03243e"         
+        projectId: "5f59f053a94acd5aa0e4ebeb"         
     },
     {
-        title: "title2",
+        title: "London Bridge",
         riskId: "riskId2",
-        description:"discription2",
+        description: "Road repair",
```

Comment:
This should be a risk description, not an activity. E.g. risk of worksite staff being struck by traffic during maintenance